app/controllers/api/v1/items_controller.rb

```diff
@@ -5,12 +5,12 @@ def index
   end
 
   def show
-    respond_with Item.find(params[:id])
+    respond_with Item.find(item_params[:id])
```

Comment:
i think we can just find by the id since it's part of the path and doesn't need to be part of the strong params